README.md

```diff
@@ -58,24 +58,27 @@ For more details, refer to [cluster_up_down.md][].
 
   - see [Fedora32.md][]
 
+- **Note about Docker Hub (docker.io image registry)**
+
+  - You *must* have a [Docker Hub] account of some kind for pulling necessary
+    images. Depending on how you use OSBS Box, you *might* require a paid account,
+    as you *might* hit the free account's hourly limit on image pulls.
+  - Consult [dockerhub.md][] for more details.
+
 ### Deployment steps
 
 1. If you haven't already, `git clone` this repository
-1. Take a look at [group_vars/all.yaml][]
+1. Take a look at [group_vars/all.yaml][][^1]
+1. You **MUST** override the 'docker_*' group vars in a YAML file, and provide
```

Comment:
https://docs.docker.com/docker-hub/access-tokens/  would you mind to mention to use access tokens here rather than password?

Reply:
Second paragraph in [dockerhub.md][]

[dockerhub.md]: https://github.com/containerbuildsystem/osbs-box/blob/fb9acaf00b116ec37ac2f47beed01d468fe11599/dockerhub.md

Reply:
Heh, I missed the whole doc in gitdiff :D